src/rgw/rgw_rest_swift.h

```diff
@@ -43,7 +43,7 @@ class RGWListBuckets_ObjStore_SWIFT : public RGWListBuckets_ObjStore {
     return 0;
   }
 public:
-  RGWListBuckets_ObjStore_SWIFT() : need_stats(true) {}
+  RGWListBuckets_ObjStore_SWIFT() : need_stats(false) {}
```

Comment:
I'm afraid that the `stats` parameter is our private extension. Tempest doesn't use it but require the stats to be always present:
```
$ sudo ngrep -W byline port 8000 -d lo

...

########
T 127.0.0.1:33146 -> 127.0.0.1:8000 [AP]
GET /v1/KEY_9775d1b07dc245849dfddf3f64857b34/?format=json HTTP/1.1.
Host: 127.0.0.1:8000.
Accept-Encoding: identity.
connection: close.
X-Auth-Token: gAAAAABZS9cb7Oh41tfiRucOVG88cS7rGnfcgmcz9siBlVZPkoKx5k76vcVbMmMcdc_RdRyAyLrwPe7R5BMUhyh0p5v9HbtisHkZ-rkClWqAT0rK35lr-AilpefEU3r916ajnHpjBDSQ9LSqkj7If3cNAB3Hg_RG458Dqh-8iQXwbMFvSnKca7g.
.

##
T 127.0.0.1:8000 -> 127.0.0.1:33146 [AP]
HTTP/1.1 200 OK.
Content-Length: 196.
X-Timestamp: 1498142492.21979.
X-Account-Container-Count: 6.
X-Account-Object-Count: 0.
X-Account-Bytes-Used: 0.
X-Account-Bytes-Used-Actual: 0.
X-Account-Access-Control: {}.
X-Trans-Id: tx00000000000000000002d-00594bd71c-106b-default.
X-Openstack-Request-Id: tx00000000000000000002d-00594bd71c-106b-default.
Accept-Ranges: bytes.
Content-Type: application/json; charset=utf-8.
Date: Thu, 22 Jun 2017 14:41:32 GMT.
Connection: close.
.

##
T 127.0.0.1:8000 -> 127.0.0.1:33146 [AP]
[{"name":"tempest-a--501653990"},{"name":"tempest-b--1185429765"},{"name":"tempest-c--475862726"},{"name":"tempest-d--118749091"},{"name":"tempest-e--1032378481"},{"name":"tempest-f--1618574056"}]
########
```